.solhint-test.json

```diff
@@ -9,7 +9,7 @@
       "max-line-length": ["error", 120],
       "max-states-count": ["warn", 30],
       "modifier-name-mixedcase": ["error"],
-      "private-vars-leading-underscore": ["error"],
+      "private-vars-leading-underscore": "off",
```

Comment:
hmm why do we need to change this?

Reply:
oh tests, yeah those have been annoying

Reply:
Can we fix the tests instead?

Reply:
Rolled back the styling change, and fixed the tests